File: experiments/runs/2026-09-17_00-24-43_claim-office-example-mapping_exact-ptdd-v1.1-refactor-subagent-cc_opus-5-no-thinking/src/item.ts
/**
 * A thing a customer asks MHPCO to insure, as the office records it on a
 * policy: what kind of thing it is, and the properties the office's rulings
 * turn on.
 */
export interface Item {
  readonly type: string;
  readonly material?: string;
  readonly cursed?: boolean;
  readonly enchantment?: number;
}

File: experiments/runs/2026-09-20_10-25-20_claim-office-java-example-mapping_exact-ptdd-v1.1-refactor-subagent-cc_opus-5-no-thinking/src/main/java/Item.java
/**
 * An item a customer wishes to insure. Components such as runes and
 * moonstones carry no material, enchantment or curse.
 */
public record Item(String type, String material, int enchantment, boolean cursed) {

    public Item(String type) {
        this(type, null, 0, false);
    }
}
